ML_experiments.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+from skimage.util.shape import view_as_windows
+
+def create_train_test_validation(dictionary, pos_key, neg_key, percent_train, percent_test, percent_validation):
+    """
+        Description: creates a training, testing, and validation dictionary for binary data  
+
+        Args:
+            dictionary -- dict (Dictionary of data sorted into pos_key and neg_key: 
+                                dictionary = {pos_key: [data, data,...], neg_key: [data, data, ...]})
+            pos_key -- String (Key for the positive data samples in dictionary)
+            neg_key -- String (Key for the negative data samples in dictionary)
+            percent_train -- double ([0-1] Percentage of data to go to training set)
```

Comment:
Should these be out of 100 if they are percentages?

Reply:
I am using them as like fractions from 0-1 for each "percentage" but I should change the wording so that's more clear that its the fraction instead of the percentage from 0-100